docsrc/markdown/configuration.md

```diff
@@ -126,3 +137,21 @@ the appId, the private key or certificate for authenticating the extension app.
 - `datafeed` contains information about the datafeed service that the bot will use for the `DatafeedLoop` service.
 If the version field is configured to `v2`, the datafeed service v2 will be used. Otherwise, the datafeed service v1 
 will be used by default.
```

Comment:
Even if you show it in the example above, please state that we can have a retry configuration specific to datafeed and explain with which default values.